deployments/server/values.yaml

```diff
@@ -158,6 +158,47 @@ requestRouting:
   # model has not yet been loaded on any cluster.
   enableDynamicModelLoading: true
 
+# Settings for rate-limiting a inference request.
+# This limiter uses GCRA. For more information, see [Generic cell rate algorithm](https://en.wikipedia.org/wiki/Generic_cell_rate_algorithm).
+# +docs:property
+rateLimit:
+  # Specify whether to enable the rate limit.
+  enable: true
+
+  # The backend store for the rate limit.
+  # NOTE: The memory is for testing purposes. When the pod restarts,
+  # the rate limit resets. It doesn't support multiple replicas.
+  # +docs:enum=memory,redis
+  storeType: memory
+
+  # Allowed request per the period.
+  # +docs:type=number
+  rate: 200
+  # The time period for the rate.
+  period: 10m
+  # The maximum burst requests.
+  # +docs:type=number
+  burst: 30
+
+  # Specify the Redis settings. This field is only used when the
+  # storageType is redis.
+  redis:
+    # The address of the Redis server.
+    address: redis-master.redis:6379
+
+    # Optional username of the Redis server.
+    # +docs:property
+    # username: ""
+
+    # The secret name that is mounted to the pod.
+    # +docs:property
+    # secretName: redis
```

Comment:
not too sure about the convention, but shouldn't we comment this out if this is not an optional field? Or should we make the field optional?